src/lib/util.ts

```diff
@@ -159,3 +159,25 @@ export function inputFieldValueAsNumber(field: any): number {
 export function inputFieldCheckedAsNumber(field: any): number {
   return Number((field as HTMLInputElement).checked);
 }
+
+/**
+ * Creates a promise that resolves when the user types the konami code.
+ */
+export function konami(): Promise<void> {
+  return new Promise((resolve) => {
+    // Keycodes for: ↑ ↑ ↓ ↓ ← → ← → B A
+    const expectedPattern = '38384040373937396665';
+    let rollingPattern = '';
+
+    const listener = (event: KeyboardEvent) => {
+      rollingPattern += event.keyCode;
+      rollingPattern = rollingPattern.slice(0, expectedPattern.length);
```

Comment:
This is wrong. It'll only look at the first 12 key presses in the "rolling pattern", thus ignoring everything after the 12th keypress.

I believe it should be `rollingPattern = rollingPattern.slice(-expectedPattern.length);` - so you get the trailing 12 keypresses.

Reply:
🔥 🔥 🔥

Reply:
lol I can't code

Reply:
(that explains a couple of issues I saw that I thought were focus issues)

Reply:
https://github.com/GoogleChromeLabs/squoosh/commit/602d5140f9e5d26f34b696852d01b6d0088e1ad7 fixed! Thanks for spotting that.